### ISSUE TITLE
Deprecation issue and its solution

### ISSUE THREAD

**Issue by hamada147** [SwiftyBeaver/AES256CBC#24]
Dear all,
Kindly notice the following. As of Swift 4 the "characters" is deprecated. 
In order to accomplish the same thing without using the deprecation element please transfer it to the following.

I'm talking about AES256CBC class in method encryptString
the following line
```swift
if !str.isEmpty && password.characters.count == 32
```
should be replaced with 
```swift
if !str.isEmpty && password.count == 32
```

and in the same class in method decryptString
the following line
```swift
if str.characters.count > 16 && password.characters.count == 32 
```
should be replaced with 
```swift
if str.count > 16 && password.count == 32 
```

**Comment by skreutzberger** [comment]
thanks, good point! we already converted the main SwiftyBeaver repo and will convert the AES256CBC repo, too in the next days.

**Comment by skreutzberger** [comment]
PS: you can create a pull request in case you want to contribute that small change by yourself ;)

**Comment by skreutzberger** [comment]
Fixed with version 1.3.1
